src/components/Calculator.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import TextBox from './TextBox';
+import FileComparison from './FileComparison';
+import { CalculatorContainer, SolidDivider, DashedDivider } from './Calculator.style';
+
+export default function Calculator(): JSX.Element {
+	return (
+		<CalculatorContainer>
+			<TextBox units={['KB', 'MB', 'GB']}></TextBox>
+			<SolidDivider></SolidDivider>
+			<TextBox units={['KB', 'MB', 'GB']}></TextBox>
+			<DashedDivider>
+				<FileComparison></FileComparison>
```

Comment:
Let's configure the attributes of these correctly now (i.e. the text, even if it's not dynamic right now, and the icons).

Reply:
Those unit fields were just placeholders for rendering the dropdowns. The reason it currently says "MB", "MB", "AR" is because the `TextBox` is waiting to use `currentUnit` from the global state

But, we've just merged in the state management branch. So Javed and I will meet today and use the types to refactor the `TextBox` component to render the correct values from the state. We won't be handling any dynamic changing of the values in this branch still though

Reply:
[Added ](https://github.com/ardriveapp/price-calculator/pull/4/commits/979d256709cce09602b79dc870b4b695868e6bd9)